wincrypto/src/dtls.rs

```diff
@@ -423,15 +413,15 @@ impl Dtls {
             e => {
                 // Failed
                 self.state = EstablishmentState::Failed;
-                Err(WinCryptoError(format!("DTLS handshake failure: {:?}", e)).into())
+                Err(format!("DTLS handshake failure: {:?}", e).into())
```

Comment:
can't you do {e:?} here?

Reply:
actually.. I can just convert the hresult to a WinCryptoError.